cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/metrics/MetricsQueryTestRun.java

```diff
@@ -211,7 +212,7 @@ public void testFlowAndAppLevelMetricsAggregation() throws Exception {
     testSingleMetric(flowletMetric2, 2);
     testSingleMetric(flowMetric, 4);
     testSingleMetric(mrMetric, 10);
-    testSingleMetric(appMetric, 14);
+    testSingleMetric(appMetric, 10);
```

Comment:
we do a gauge at mr level in the end , so that should be the latest value when we query for reads at app level.